### ISSUE TITLE
Alt text wrong typo & copyright year is hardcoded must be fixed to dynamic

### ISSUE THREAD

**Issue by o000SAI000o** [InnoWebLabs/Vehigo#208]
1. The alt text "Toyota Innova Crsyta" contains a typo. It should be "Toyota Innova Crysta" 
2. The copyright year is hardcoded as 2023. Consider updating it dynamically or ensuring it is current.
